opensrp-anc/src/main/java/org/smartregister/anc/library/util/ConstantsUtils.java

```diff
@@ -240,4 +240,8 @@
     public static class ClientUtils {
         public static final String ANC_ID = "ANC_ID";
     }
+
+    public static class SettingsSyncParams{
```

Comment:
Codacy found an issue: [The utility class name 'SettingsSyncParams' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)